packages/datagrid/stories/datagrid.component.js

```diff
@@ -4,11 +4,37 @@ import { checkA11y } from '@storybook/addon-a11y';
 import { IconsProvider } from '@talend/react-components';
 
 import DataGrid from '../src/components/';
+import DefaultRenderer from '../src/components/DefaultCellRenderer/DefaultRenderer.component';
+import DefaultBooleanCellRenderer from '../src/components/DefaultBooleanCellRenderer';
+import DefaultIntCellRenderer from '../src/components/DefaultIntCellRenderer';
+import DefaultPinHeaderRenderer from '../src/components/DefaultPinHeaderRenderer';
+import DefaultCellRenderer from '../src/components/DefaultCellRenderer';
+import DefaultHeaderRenderer from '../src/components/DefaultHeaderRenderer';
+
 import serializer from '../src/components/DatasetSerializer';
 import sample from './sample.json';
 import sample2 from './sample2.json';
 import sample3 from './sample3.json';
 
+function getComponent(component) {
+	switch (component) {
+		case 'DefaultIntCellRenderer':
+			return DefaultIntCellRenderer;
+		case 'DefaultBooleanCellRenderer':
+			return DefaultBooleanCellRenderer;
+		case 'DefaultHeaderRenderer':
+			return DefaultHeaderRenderer;
+		case 'DefaultPinHeaderRenderer':
+			return DefaultPinHeaderRenderer;
+		case 'DefaultCellRenderer':
+			return DefaultCellRenderer;
+		case 'DefaultStringCellRenderer':
+			return DefaultRenderer;
+		default:
+			console.error(component);
+	}
+}
```

Comment:
This can become a very big switch case in the future if we have exotic need / feature...

Reply:
it is only for the story.